index.rst

```diff
@@ -92,53 +138,106 @@ The electometer sits in the electronics box and the cable from the photodiode is
 
 Information on the electrometer and photodiode can be found on Docushare `here <https://docushare.lsst.org/docushare/dsweb/View/Collection-5176>`__
 
+
 Optics
 ------
 There are three optical that deliver the beam from the monochromator to the calibration screen. The first is a cylindrical lens (CKX20) that shapes the beam and can be rotated. The light then goes to an asphere (Thorlab ACL7560U-A) and then to a fold mirror that reflects the light straight up through the shutter. The optical design for this system can be found on `Docushare <https://docushare.lsst.org/docushare/dsweb/View/Collection-11943>`__.
```

Comment:
Made add a sentence that the optical design is being re-done.

Reply:
done